roles/lib_openshift/library/oc_route.py

```diff
@@ -1707,14 +1704,14 @@ def run_ansible(params, check_mode=False):
         if params['tls_termination'] and params['tls_termination'].lower() != 'passthrough':  # E501
 
             for key, option in files.items():
-                if key == 'destcacert' and params['tls_termination'] != 'reencrypt':
+                if not option['path'] and not option['content']:
```

Comment:
two things

1) modifying lib_openshift should be in it's own PR
2) when modifying lib_openshift, you need to modify lib_openshift/src/* files and then generate the lib_openshift/library/* files